build/Microsoft.DotNet.Cli.BundledTemplates.proj

```diff
@@ -0,0 +1,42 @@
+<Project ToolsVersion="15.0" DefaultTargets="CopyTemplateToOutput">
+  <!-- workaround for https://github.com/Microsoft/msbuild/issues/885 -->
+  <!-- renaming the property because the original property is a global property and therefore
+       cannot be redefined at runtime. -->
+  <PropertyGroup>
+    <CLIBuildDllPath>$([MSBuild]::Unescape($(CLIBuildDll)))</CLIBuildDllPath>
+  </PropertyGroup>
+
+  <UsingTask TaskName="DotNetRestore" AssemblyFile="$(CLIBuildDllPath)" />
```

Comment:
Can't you use the tasks already defined at https://github.com/dotnet/cli/blob/rel/1.0.0/build/Microsoft.DotNet.Cli.tasks?